index.html

```diff
@@ -124,6 +124,22 @@ <h2>Доп. Информация о расположении и контакты
       <a href="https://htmlacademy.ru">HTML Academy</a>
     </p>
   </footer>
+  <form class="#" action="#" method="post">
+    <button type="button">#</button> <!-- Кнопка закрыть-->
+    <p>
+      <label for="first_and_second_name">Ваше имя:</label>
+      <input id="first_and_second_name" type="text" name="first_and_second_name" value="" placeholder="Имя Фамилия" />
+    </p>
+    <p>
+      <label for="customer_email">Ваш e-mail:</label>
+      <input id="customer_email" type="text" name="customer_email" value="" placeholder="email@example.com" />
+    </p>
+    <p>
+      <label for="message_text">Текст письма:</label>
+      <input id="message_text" type="text" name="message_text" value="" placeholder="В свободной форме" />
```

Comment:
```suggestion
      <textarea id="message_text" name="message_text" placeholder="В свободной форме"></textarea>
```
Когда ты видишь поля формы длиннее, чем одна строка - это текстареа: https://www.w3.org/TR/html52/sec-forms.html#the-textarea-element